src/services/syntaxServices.js

```diff
@@ -0,0 +1,19 @@
+services.factory('syntaxServices', [function() {
+  
+	function validateNifSyntax(nif) {
```

Comment:
This function is for KLM nif. I think there should be another one for legal NIFs.